frappe/templates/web.html

```diff
@@ -17,7 +17,7 @@
 	</div>
 
 	{% block page_container %}
-	<main class="{% if not full_width %}container my-5{% endif %}">
```

Comment:
Where is this margin compensated? Is it on `.page-content-wrapper` ?

Reply:
Not exactly, this reduces the large space between the breadcrumb and the main content only. 

The changes on `.page-content-wrapper` are to level it with the sidebar